tests/functional/testMultihop.js

```diff
@@ -2,639 +2,690 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-const assert = require('assert');
-const queries = require('./queries.js');
-const vpn = require('./helper.js');
-
-/* TODO
-async function selectCityFromList(cityId, countryId) {
-  await vpn.setQueryProperty(
-      COUNTRY_VIEW, 'contentY',
-      parseInt(await vpn.getElementProperty(cityId, 'y')) +
-          parseInt(await vpn.getElementProperty(countryId, 'y')));
-}
-
-async function selectCountryFromList(countryId) {
-  await vpn.setQueryProperty(
-      COUNTRY_VIEW, 'contentY',
-      parseInt(await vpn.getElementProperty(countryId, 'y')));
-}
-*/
-
-describe('Server list', function() {
-  let servers;
-  let currentCountryCode;
-  let currentCity;
-
-  this.timeout(240000);
-  this.ctx.authenticationNeeded = true;
-
-  beforeEach(async () => {
-    await vpn.waitForQueryAndClick(queries.screenHome.SERVER_LIST_BUTTON);
-    await vpn.waitForQuery(queries.screenHome.STACKVIEW.ready());
-
-    servers = await vpn.servers();
-    currentCountryCode = await vpn.getMozillaProperty(
-        'Mozilla.VPN', 'VPNCurrentServer', 'exitCountryCode');
-    currentCity = await vpn.getMozillaProperty(
-        'Mozilla.VPN', 'VPNCurrentServer', 'exitCityName');
-
-    for (let server of servers) {
-      if (currentCountryCode === server.code) {
-        for (let city of server.cities) {
-          if (city.name == currentCity) {
-            currentCity = city.localizedName;
-            break;
+ const assert = require('assert');
```

Comment:
why this extra space before each line? It makes almost impossible to review this PR. Can you please format this file correctly?